content/en/graphing/_index.md

```diff
@@ -140,51 +115,9 @@ To display only your formula, un-check your metrics **a** and **b**:
-|-----------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `Min`/`Max`           | No       | Specify the minimum and / or maximum value to show on y-axis. It takes a number or `Auto` as the default value.                                                                                                   |
-| `Scale`               | No       | Specifies the scale type. Possible values:<br>- *linear*: A linear scale (default)<br>- *log*: A logarithmic scale<br>- *pow*: A Power of 2 scale (2 is default, modify in JSON)<br>- *sqrt*: A square root scale |
-| `Always include zero` | No       | Always include zero or fit the axis to the data range. The default is to always include zero.                                                                                                                     |
-
-**Note**: Because the mathematical log function doesn't accept negative values, the Datadog log scale only works if values are of the same sign (everything > 0 or everything < 0). Otherwise an empty graph is returned.
-
-### Overlay events for additional context
-
-Add events from related systems to add more context to your graph. For example, you can add GitHub commits, Jenkins deploys, or Docker creation events. Expand the **Event Overlays** section and enter a query to display those events. Use the same query format as for the [Event Stream][8], for example:
-
-| Query                       | Description                                                |
-|-----------------------------|------------------------------------------------------------|
-| `sources:jenkins`           | Shows all events from the Jenkins source.                  |
-| `tag:role:web`              | Shows all events with the tag `role:web`.                  |
-| `tags:$<TEMPLATE_VARIABLE>` | Shows all events from the selected [Template Variable][9]. |
-
-{{< img src="graphing/index/overlay_events.png" alt="Overlay Events" responsive="true" style="width:75%;" >}}
-
 ## Create a title
 
-If you don't enter a title, one is automatically generated based on your selections. But it may be more useful to the users of the [dashboard][10] to create a title that more aptly describes the purpose of the graph. Linking the technical purpose to the business benefits adds even more value.
+If you don't enter a title, one is automatically generated based on your selections. But it may be more useful to the users of the [dashboard][8] to create a title that more aptly describes the purpose of the graph. Linking the technical purpose to the business benefits adds even more value.
```

Comment:
```suggestion
If you don't enter a title, one is automatically generated based on your selections. However, it is recommended that you create a title that aptly describes the purpose of the graph. 
```
"adding value" sounds kind of marketing. it should be evident to the user that they should title stuff well.